recursive_iterativeOddSum.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+function recursiveOddSumTo(number) {
+    let sum = 1;
+    let currentOddNumber = 1;
+    let limitNumber = number;
+    number % 2 === 0 ? limitNumber -= 1 : limitNumber -= 2;  

```

Comment:
why do you need this line?

Reply:
Long story. Put it plainly - I thought the given number (if odd) is not included in the range. To get this result I used this line. Now I see I was wrong, it is included in the range. I have rewritten the function.